### ISSUE TITLE
Error On User Model delete - Integrity constraint Violation

### ISSUE THREAD

**Issue by JonathanPort** [musonza/chat#109]
SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`my-database`.`mc_messages`, CONSTRAINT `mc_messages_user_id_foreign` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`)) (SQL: delete from `users` where `id` = 39)

**Comment by JonathanPort** [comment]
Thanks for the update @musonza 👍🏼

If you're looking for a PHP dev to help out with anything, even the smallest task, I'm happy to help! I want to gain some experience working along side other devs with packages like this and would be proud to contribute. 

Email me hello@jonathanport.com if you need anything!

Cheers,
Jon

**Comment by musonza** [comment]
If you want to look at this https://github.com/musonza/groups . Haven't updated it in a while and it can use some improvements

**Comment by JonathanPort** [comment]
Yeah @musonza Will do! I'll have look through it. Do you have Twitter or an email address we can chat with? (Email me the details if you don't want to publicly share)

**Comment by musonza** [comment]
yes i sent you an email to that email above before my last comment.
